### ISSUE TITLE
struct2depth Disparity images look weird during training

### ISSUE THREAD

**Issue by aj96** [tensorflow/models#6774]
### System information
- **What is the top-level directory of the model you are using**: struct2depth
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 18.04
- **TensorFlow installed from (source or binary)**: pip install tensorflow-gpu
- **TensorFlow version (use command below)**: 1.13.1
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: 10.0 / 7.4.1
- **GPU model and memory**: Tesla-P100 16 Gb
- **Exact command to reproduce**: Run the training script

I have been training struct2depth for a while (on the 25th epoch). The total loss is going down over time, but I am seeing some very weird disparity images during training. They make no sense and do not seem to align with the input images.

<img width="884" alt="weird_disparity_images_during_training" src="https://user-images.githubusercontent.com/19962279/57641134-153f5a00-7569-11e9-89dc-423261324171.png">

Can anyone comment on this? Thank you.

**Comment by ymodak** [comment]
@aneliaangelova Can you please take a look? Thanks!

**Comment by aneliaangelova** [comment]
I have not seen such outputs from the model although the strange depth image (shown twice, but maybe offsetted a bit in time) seem to correspond to the "blue wall" image (which is not great but possible as the image is hard) and not to the yellow building image as the snapshot seem to indicate. The middle column images do not match to each other too.
Can you please look into the data and make sure this does not happen on only isolated (and hard to predict scenes)?

**Comment by aj96** [comment]
It seems like the images are almost always offset in time. Is there any particular reason for this? And no, it doens't happen on all the images. But it is very frequent.  Here is another example.

<img width="884" alt="weird_disparity_images_during_training" src="https://user-images.githubusercontent.com/19962279/57752020-70259e00-769c-11e9-96e5-7bf7616d2cbc.png">

Also, the default number of batch steps for training is 10,000,000. At 150s per 100 batch steps, this would take about 173 days to train. Is this correct?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
